pwa/pages/about/index.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { Subtitle, Title, Text } from '../../components/common/text';
+import Link from 'next/link';
+import { NextPage } from 'next';
+
+type AboutPropsType = {
+  type: AllowedElement;
+  text?: string;
+  elements?: string[];
+};
+
+type AllowedElement = 'title' | 'subtitle' | 'text' | 'list';
+
+function mapTextComponent({ type, text, elements }: AboutPropsType, index: number): React.ReactElement | undefined {
+  switch (type) {
+    case 'title':
+      return <Title title={text || ''} key={index} />;
+    case 'subtitle':
+      return <Subtitle text={text || ''} key={index} />;
+    case 'text':
+      return <Text text={text || ''} key={index} />;
+    case 'list':
+      return (
+        <ul className="list-disc pl-8" key={index}>
+          {elements && elements.map((element, idx) => <li key={idx}>{element}</li>)}
+        </ul>
+      );
+    default:
+      return undefined;
+  }
+}
```

Comment:
Explain why you need that.

Reply:
I thoug it was better to map on object for more readability in the code. I thought : { type: 'text', text: 'Type de site : Saas' }, was better than writing a huge number of time : <h3 className="...">...</h3>

Reply:
You're creating some compute on an ultra static page. That doesn't make sense to map over an object when the page won't change often.